views/component/list/template.njk

```diff
@@ -3,12 +3,12 @@
     {% for Course in pageResults.results %}
 
     <li class="list-details u--border--bgrey">
-        <a class="govuk-link list-title" href="#">{{ Course._title }}</a>
+        <a class="govuk-link list-title" href="/course/{{ Course._id }}">{{ Course._title }}</a>
         {% if Course._modules.length > 1 %}
             <p class="govuk-body">This course comprises of {{Course._modules.length}} modules</p>
         {% endif %}
         <p class="govuk-body">{{ Course._shortDescription }}</p>
-        <a class="list-link list-padding" href="#">Edit</a>
+        <a class="list-link list-padding" href="/course/{{ Course._id }}">Edit</a>
```

Comment:
This Href will change to a different path once we get the edit template avaialble